src/game/server/gamemodes/mod.cpp

```diff
@@ -432,6 +433,30 @@ void CGameControllerMOD::Tick()
 		
 		m_RoundStartTick = Server()->Tick();
 	}
+
+	if (m_ReportTick <= 0)
+	{
+		CPlayerIterator<PLAYERITER_ALL> Iter(GameServer()->m_apPlayers);
+		
+		dynamic_string Buffer;
+
+		Buffer.append("online list=[");
+		if (Iter.Next())
+		{
+			Buffer.append(Server()->ClientName(Iter.Player()->GetCID()));
+		}
+
+		while(Iter.Next())
+		{
+			Buffer.append(", ");
+			Buffer.append(Server()->ClientName(Iter.Player()->GetCID()));
+		}
+		Buffer.append("]");
+		GameServer()->Console()->Print(IConsole::OUTPUT_LEVEL_DEBUG, "game", Buffer.buffer());
```

Comment:
Does it mean that if i compile the server not as server_sql_debug, but as server_sql, this output will not be visible?

Reply:
I couldn't quite follow what your intention is but yes, I've just tested this:
Using `bam conf=server_sql` will still display a list that is for example: 
```
[game]: online list=[ArmadilIo]
```

Reply:
Cool, thanks. My intention was to be sure that if i recompile the server without debug, the log entries are still there.